backend/app/api/endpoints/devices.py

```
@@ -0,0 +1,44 @@
+from app.api.dependencies import get_db
+from app.core import security
+from app.models.user import User
+from app.schemas.device import DeviceInfo, DeviceListResponse
+from app.services.device_service import device_service
+
+router = APIRouter()
+
+
+@router.get("/online", response_model=DeviceListResponse)
+async def get_online_devices(
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)